pkg/runtime/cli.go

```diff
@@ -148,3 +150,17 @@ func FromFlags() (*DaprRuntime, error) {
 	}
 	return NewDaprRuntime(runtimeConfig, globalConfig), nil
 }
+
+// getDefaultComponentsFolder returns the hidden .components folder created at init time
+func getDefaultComponentsFolder() string {
```

Comment:
nit: can we rename getDefaultComponentsFolder to `defaultComponentPath` ? Folder is the term from Windows.

Reply:
fixed